composer.json

```diff
@@ -109,6 +109,9 @@
             },
             "drupal/socialbase": {
                 "Issue #3473257: Improve the reset password link according to the accessibility changes": "https://www.drupal.org/files/issues/2024-09-10/3473257-reset_password_improvements-1.patch"
+            },
+            "drupal/socialblue": {
+                "Issue #3475568: Add hyperlink to Expertise and Interests profile tags": "https://www.drupal.org/files/issues/2024-09-19/3475568-4-add-hyperlink-to-expertise-interest-profile-tags.patch"
```

Comment:
Curious why you chose to do this in the template file of the theme.
Imagine our users having a different subtheme, that means these will not be linked. While I would imagine that this is more of rendering decisions, where we say this label renders as a link leading to the search, regardless of which theme our community uses.

Reply:
Hey Ronald,
The JIRA opened by Jake has another two fields with hyperlink to be used as example, so I followed the pattern from these fields.
References: 
- https://git.drupalcode.org/project/socialblue/-/blob/2.6.x/templates/profile/profile--profile--default--sky.html.twig?ref_type=heads#L131
- https://git.drupalcode.org/project/socialblue/-/blob/2.6.x/templates/profile/profile--profile--default--sky.html.twig?ref_type=heads#L161

Reply:
Hi @vcsvinicius, thank you!
Gotcha, so I think, from a rendering perspective it should likely not be necessary to have these rendered as anchors in twig. You'd probably be able to do that just using the render array.
As of now I can approve it, perhaps we can think of using a follow up to see if we rather have all those instances fixed or not, or if that isn't priority. 👍 

Would you be able to get a new release for Socialblue planned with Denis?